docs/pages/guides/mesheryctl/authenticate-to-mesheryctl.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Authenticating to `mesheryctl` 
+permalink: guides/mesheryctl/authenticate-to-mesheryctl
+language: en
+type: Guides
+category: mesheryctl
+---
+
+To authenticate to mesheryctl you have to use the command `mesheryctl system login`. On execution of this command you get a an option to choose a provider, you would have to choose `Meshery` as your provider to authenticate yourself. After doing so, a new window opens up with ways to login. 
+
+## Get your Token
+
+##### There are 2 ways to get your token:
+
+- Get your token through **Meshery Dashboard**, from the `Get Token` option.
+
+    _Downloading the token_
+
+    <a href="{{ site.baseurl }}/assets/img/token/token.png"><img alt="Meshery Dashboard" src="{{ site.baseurl }}/assets/img/token/token.png" /></a>
+    <br/>
+    <br/>
+
+- Get your token through **Meshery CLI**.
+    <br/>
+   > To get the token through `mesheryctl` you would have to use the following command and the path to token for authenticating to Meshery API (default "auth.json").
+    <br/>
+    <pre class="codeblock-pre">
+        <div class="codeblock">
+        mesheryctl system config --token [path-to-file]
+        </div>
+    </pre>
+
+
+## Need for Authentication
+
+The basic need for authentication to `Meshery` provider is:
+- Persistent Sessions
+- Save environment setup
+- Retrieve performance test results 
+
+
+# Suggested Reading
```

Comment:
Use an {% include %} for this section.

Reply:
okay, ill work on making a {% include %} for this section.

Reply:
Thank you much.

Reply:
> @leecalcote one question that I had in mind was different pages have different `Suggested Reading` so how exactly would we make an _includes file for this.🤔 // cc @alphaX86

I never used `_includes` for one time in my works... But I'll try to figure out and get back to you

Reply:
Thank you🙏😅 @alphaX86

Reply:
@warunicorn19 I just got a mere info for that includes... well you need to make a HTML page instead of markdown and then use it to import that, just like iframes

Reply:
> > @leecalcote one question that I had in mind was different pages have different `Suggested Reading` so how exactly would we make an _includes file for this.🤔 // cc @alphaX86
> 
> I never used `_includes` for one time in my works... But I'll try to figure out and get back to you

There are couple ways of doing this: passing in variables and/or front matter variables as context-specific filters for queries that you might be using against datafile or collections.